src/main/cli.ts

```diff
@@ -0,0 +1,98 @@
+import { KANGAROO_CONFIG } from './const';
+import { breakingAppVersion } from './filesystem';
+import { app } from 'electron';
+
+export interface CliOpts {
+  profile?: string;
+  networkSeed?: string | undefined;
+  holochainPath?: string | undefined;
+  lairPath?: string | undefined;
+  holochainRustLog?: string | undefined;
+  holochainWasmLog?: string | undefined;
+  lairRustLog?: string | undefined;
+  bootstrapUrl?: string;
+  signalingUrl?: string;
+  iceUrls?: string;
+  printHolochainLogs?: boolean;
+}
+
+export interface RunOptions {
+  profile: string | undefined;
+  networkSeed: string;
+  bootstrapUrl: string | undefined;
```

Comment:
I'd be tempted to make these a URL type to be as defensive as possible about the inputs to the CLI. Not necessary but nice to be told early that your inputs are invalid

Reply:
I think that's a good idea. Otherwise the error is thrown somewhere in holochain which is nasty to interpret. The URL will need to be converted back to a string again when writing the conductor config but it's worth as a format validation layer here anyway I think.